app/models/concerns/courses/edit_options/subjects_concern.rb

```diff
@@ -8,7 +8,11 @@ module SubjectsConcern
         #
         # https://github.com/DFE-Digital/manage-courses-frontend/blob/master/spec/factories/courses.rb
         def potential_subjects
-          self.assignable_master_subjects
+          if level == "secondary"
```

Comment:
When not sort subjects regardless?

Reply:
Yea good shout. Previously had it that way 🤦‍♂